server/routes/notification.js

```diff
@@ -1,5 +1,6 @@
 const router = require('express').Router()
 const { check } = require('express-validator')
+const moment = require('moment')
```

Comment:
I thought `moment` was now deprecated and shouldn't be used for projects?

Reply:
agreed, Ill use date-fns instead